packages/camera/camera/lib/src/camera_controller.dart

```diff
@@ -480,13 +480,14 @@ class CameraController extends ValueNotifier<CameraValue> {
   /// Throws a [CameraException] if image streaming or video recording has
   /// already started.
   ///
-  /// The `startImageStream` method is only available on Android and iOS (other
-  /// platforms won't be supported in current setup).
+  /// The `startImageStream` method is only available on Android, iOS and
+  /// Windows (other platforms won't be supported in current setup).
```

Comment:
See https://github.com/flutter/packages/pull/7220#issuecomment-2314872164. This PR should include adding an API to query support, rather than modifying the legacy approach of hard-coding platforms.

Reply:
As far as I can tell, Windows is the sole outlier without image streaming support. Perhaps we could just eliminate the check altogether?

Reply:
`camera_web` does not support streaming. There may also be third-party, unendorsed implementation that don't support streaming.

Reply:
Opened #8250 to get started.

Reply:
See https://github.com/flutter/flutter/blob/master/docs/ecosystem/contributing/README.md#changing-federated-plugins